toolkit/tools/pkg/imagecustomizerlib/artifactsinputoutput.go

```diff
@@ -270,68 +278,75 @@ func writeInjectFilesYaml(metadata []imagecustomizerapi.InjectArtifactMetadata,
 	return nil
 }
 
-func InjectFilesWithConfigFile(ctx context.Context, buildDir string, configFile string, inputImageFile string,
-	outputImageFile string, outputImageFormat string,
-) error {
-	var injectConfig imagecustomizerapi.InjectFilesConfig
-	err := imagecustomizerapi.UnmarshalYamlFile(configFile, &injectConfig)
+func InjectFilesWithConfigFileOptions(ctx context.Context, configFile string, options InjectFilesOptions) error {
```

Comment:
You can probably keep the `InjectFilesWithConfigFile` and `InjectFiles` names. I created wrapper functions for the `CustomizeImage` function since I didn't want to change the large number of existing tests using the existing function names.